lib/si-frontend-types-rs/src/fs.rs

```diff
@@ -196,9 +196,9 @@ pub enum Binding {
     CodeGeneration {
         inputs: Vec<LeafInputLocation>,
     },
-    Management {
-        managed_schemas: Option<Vec<String>>,
-    },
+    // TODO can we remove this? Don't want to remove directories if it's necessary, and I don't
```

Comment:
Question for si-fs expert.

Reply:
We can remove this later, right now it's harmless to keep it in

Reply:
Removed the comment!